pkg/framework/plugins/defaultevictor/defaultevictor.go

```diff
@@ -195,6 +195,17 @@ func New(args runtime.Object, handle frameworktypes.Handle) (frameworktypes.Plug
 			return nil
 		})
 	}
+
+	if defaultEvictorArgs.IgnorePodsWithoutPDB {
+		ev.constraints = append(ev.constraints, func(pod *v1.Pod) error {
+			hasPdb, err := utils.IsPodCoveredByPDB(pod, handle.SharedInformerFactory().Policy().V1().PodDisruptionBudgets().Lister())
+			if err != nil || !hasPdb {
+				return fmt.Errorf("no PodDisruptionBudget found for pod")
```

Comment:
I can do that, in fact I did that in the initial pass on the code. The `PodDisruptionBudgetLister` implementation has a, in my opinion :-) design choice that less that ideal: https://github.com/john7doe/descheduler/blob/master/vendor/k8s.io/client-go/listers/policy/v1/poddisruptionbudget_expansion.go#L63-L65

TL;DR no pdb is returned as an error :-(

So logging errors would be quite verbose. Its also a plain error, so `errors.Is` is not an option to differentiate between actual errors and `len() == 0` ones. 

One option could be to log at a very high verbosity level, so that the errors are normally no logged, but can be seen if someone needs to investigate.

Reply:
I see. That's unfortunate. In that case it will be better to call `PodDisruptionBudgets.List` directly/copy paste some of the code from https://github.com/john7doe/descheduler/blob/master/vendor/k8s.io/client-go/listers/policy/v1/poddisruptionbudget_expansion.go#L63-L65.

Reply:
@ingvagabund done, please have a look